### PR TITLE
[DO-1858] [Contains Migration] Added model for luxury car

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/models/declaration_models.py [hunk 1]
```diff
@@ -1179,6 +1179,55 @@ class Vehicle(DataOceanModel):
...
+        blank=True,
+        help_text='engine cylinder volume'
+    )
+    fuel = models.SmallIntegerField(
+        'fuel',
+        choices=FUEL_TYPE,
+        null=True,
+        blank=True,
+        help_text='fuel type',
+    )
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1225]:
```suggestion
    fuel_type = models.SmallIntegerField(
        choices=FUEL_TYPE,
        null=True,
        blank=True,
    )
```

File: business_register/models/declaration_models.py [hunk 2]
```diff
@@ -1179,6 +1179,55 @@ class Vehicle(DataOceanModel):
...
+        help_text='year of the document in which the car is indicated',
+    )
+    volume = models.DecimalField(
+        'volume',
+        max_digits=2,
+        decimal_places=1,
+        null=True,
+        blank=True,
+        help_text='engine cylinder volume'
+    )
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1218]:
```suggestion
    volume = models.DecimalField(
        max_digits=2,
        decimal_places=1,
        null=True,
        blank=True,
        help_text='engine cylinder volume'
    )
```

File: business_register/models/declaration_models.py [hunk 3]
```diff
@@ -1179,6 +1179,55 @@ class Vehicle(DataOceanModel):
...
+        help_text='model',
+    )
+    after_year = models.PositiveSmallIntegerField(
+        'after year',
+        help_text='year of manufacture of the car after which the car is considered luxury',
+    )
+    document_year = models.PositiveSmallIntegerField(
+        'document year',
+        help_text='year of the document in which the car is indicated',
+    )
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1210]:
```suggestion
    brand = models.CharField(max_length=80)
    model = models.CharField(max_length=140)
    after_year = models.PositiveSmallIntegerField(
        help_text='year of manufacture of the car after which the car is considered luxury',
    )
    document_year = models.PositiveSmallIntegerField(
        help_text='year of the document in which the car is indicated',
    )
```

File: business_register/models/declaration_models.py [hunk 4]
```diff
@@ -1179,6 +1179,55 @@ class Vehicle(DataOceanModel):
...
+    fuel = models.SmallIntegerField(
+        'fuel',
+        choices=FUEL_TYPE,
+        null=True,
+        blank=True,
+        help_text='fuel type',
+    )
+
+    class Meta:
+        unique_together = (('brand', 'model', 'after_year'),)
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1228]:
```suggestion
        unique_together = (('brand', 'model', 'document_year'),)
```

File: business_register/models/declaration_models.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

File: business_register/models/declaration_models.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

File: business_register/models/declaration_models.py (hunk 3 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

File: business_register/models/declaration_models.py (hunk 4 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

**Review by Andrii17s — APPROVED** [review]

**Review by IegorSoboliev — APPROVED** [review]

**Pull request merged**